### ISSUE TITLE
New version, minor tweaks

### ISSUE THREAD

**Issue by bertelschmitt** [AntonMu/TrainYourOwnYOLO#182]
Hi, @AntonMu 
Just when I was (somewhat) finished  with the multi-stream, multi-GPU mod, along with a monster of a driver app to put it through its paces, you dropped the new version updated to modern Tensorflow/Keras. Thank you!  I adapted my code to it, will test for another week, and then put it up. 

Allow me to recommend two minor tweaks in the meantime: 

In Convert_to_YOLO_format.py  change  line 79 to  `file.write(str(elem[0]) + "\n")`    
It threw an error when elem[0] was an int or float.

In model.py, change lines  162-170  to
`
    box_xy = (K.sigmoid(feats[..., :2]) + grid) / K.cast(grid_shape[..., ::-1], K.dtype(feats))
    box_wh = K.exp(feats[..., 2:4]) * anchors_tensor / K.cast(input_shape[..., ::-1], K.dtype(feats))
`
Old code threw streams of warnings without.

Thank you for all you do.

B
